extensions/eclipse-che-theia-plugin-ext/webpack.config.js

```diff
@@ -22,6 +22,9 @@ module.exports = {
                 use: [
                     {
                         loader: 'ts-loader',
+                        options: {
+                            transpileOnly: true
```

Comment:
hi, why do we need this change ?

Reply:
this is temporary, in some reason got error form webpack